unicodetools/src/main/java/org/unicode/props/UnicodeProperty.java

```diff
@@ -4,7 +4,7 @@
  * others. All Rights Reserved.                                                *
  *******************************************************************************
  */
-package org.unicode.cldr.util.props;
+package org.unicode.props;
```

Comment:
Looking at the description of issue #66, we have been using the CLDR version of UnicodeProperty which has apparently changed in significant ways. (I don't remember what diffs I looked at almost a year ago...)

From the issue:
> The CLDR version is the newer one. ...
> For now, I am adding a note to the setup instructions to move cldr-code above unicodetools in the build path, ...

Now that you change the unicodetools code to explicitly use this org.unicode.props.UnicodeProperty, shouldn't this be a copy of the CLDR version that we have been using, rather than the unicodetools/org.unicode.cldr version that was shadowed, in order to keep the behavior intact?

Reply:
I'm not sure we've been using the CLDR version consistently. I think it may be hit or miss depending on how it is packaged or loaded. Some code worked, some didn't.

but… that's an interesting tack.

Reply:
I assumed the one in unicodetools was 'needed' somehow.

Reply:
Discussed in live meeting. Try to use/copy the CLDR version.

Reply:
Resolved, I think, after various experiments:
The CLDR code needs to call the CLDR version of UnicodeProperty, but the Unicode Tools code that uses UnicodeProperty directly needs to use its own version.